framework/src/executioners/Transient.C

```diff
@@ -638,6 +638,13 @@ Transient::solveStep(Real input_dt)
       _time_stepper->acceptStep();
       return;
     }
+    else if (numPicardIts() == _picard_max_its)
+    {
+      _console << "Maximum number of Picard iterations reached! Moving onto the next time step..."
+               << std::endl;
+      _time_stepper->acceptStep();
```

Comment:
Ummm - no.  This should be failure.

Reply:
Maybe I am misunderstanding what `picard_max_its` is designed to. Based on the code however, it seems to me that its intended behavior is designed to be different from  `l_max_its` and `nl_max_its` (although the naming convention would be bad) in the sense that it seems to be the maximum number of Picard iterations before moving on to the next step.

The reasons I am saying this are:
- the default value for `picard_max_its` is 1. If it was defined to be the maximum number of Picard iterations before aborting the solve, the default behavior should fail every time the tolerances are not met after the first Picard. Instead, the default behavior is not immediately go to the next time step.
- the docstring reads `Number of times each timestep will be solved` which again seems to indicate the maximum number of times a time step will be repeated before going to the next one. Either way, that docstring could probably be improved.

So my questions are:
1) What is `picard_max_its` designed to do? 
2) If it is the maximum number of iterations before aborting the solve, should the default value be changed to avoid confusion?

Reply:
Hitting `l_max_its` _doesn't_ fail the timestep because that's how inexact Newton works (you undersolve the linear system)... and it ultimately doesn't matter as long as the nonlinear system converges.

Hitting 'nl_max_its' _does_ fail the linear solve because you failed to find the answer (to a sufficient tolerance).

`picard_max_its` is the same as `nl_max_its`.... if you hit it then you have failed to find a stationary point (a solution to the whole nonlinear problem).... and it should fail the step (which it currently does) so that you can retry with a smallter dt, etc.

Not failing the staep when you hit `picard_max_its` would be odd.  It would be like saying "I know I didn't solve the problem but I want to continue anyway".  It would be somewhere between "loose coupling"  and Picard.  If that's what you want... then that needs to be a separate option/parameter.

The default for hitting `picard_max_its` must continue to be that it fails the solve.